examples/raytracing-rgen/shader.rgen

```diff
@@ -0,0 +1,30 @@
+#version 460 core
+#extension GL_EXT_ray_tracing : require
+#extension GL_EXT_ray_flags_primitive_culling : enable
+
+layout(set = 0, binding = 0) uniform accelerationStructureEXT accelStruct;
+
+layout(location = 0) in uint rayFlags;
+layout(location = 1) in uint cullMask;
+layout(location = 2) in uint offsetSBT;
+layout(location = 3) in uint strideSBT;
+layout(location = 4) in uint missIndex;
+layout(location = 5) in vec3 rayOrigin;
+layout(location = 6) in float rayTMin;
+layout(location = 7) in vec3 rayDirection;
+layout(location = 8) in float rayTMax;
+layout(location = 9) rayPayloadEXT vec3 payload;
+
+layout(location = 10) in uint indexSBTCallable;
+layout(location = 11) callableDataEXT dvec2 callableData;
```

Comment:
Is this value typically a double? The interpreter doesn't actually support doubles yet (it silently treats them as floats), so this is something for me to put higher on the to do list if so.

Also, be aware of this if you add other tests to this example or aren't seeing the precision you are expecting.

I am happy to have a real use case for double values, so please don't change this!

Reply:
Will keep doubles in mind.